state/backend.go

```diff
@@ -57,6 +59,24 @@ func (st *State) localID(id string) string {
 	return localID
 }
 
+// ParseLocalIDToTags tries to parse a DocID e.g. `c9741ea1-0c2a-444d-82f5-787583a48557:a#mediawiki
+// to a corresponding Tag. In the above case -> applicationTag.
+func (st *State) ParseLocalIDToTags(docID string) names.Tag {
```

Comment:
I know there is a precedent in `strictLocalID` below, but let's remove the receiver from this, as it is not used. Then it can be removed from the state indirections in the space API server logic and the various backings.

There is a case for this being in core, but for now put it in _state/state.go_.

Reply:
should be solved in commit 02053e3e4824ee87bc3b43e15a41cbd7e025588f